jerry-debugger/jerry-client-ws.py

```diff
@@ -312,11 +312,38 @@ def do_backtrace(self, args):
 
     def do_src(self, args):
         """ Get current source code """
-        if self.disable_args(args):
+        if args:
+            try:
+                line_num = int(args)
+                if line_num < 0:
+                    print("Error: Positive integer number expected")
+                    return
+            except ValueError as val_errno:
+                print("Error: Positive integer number expected, %s" % (val_errno))
+                return
```

Comment:
Maybe we could extract the convert logic to a helper method, so then we would not need to duplicate the code here and in the next method.